documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -153,7 +147,17 @@ A https://docs.mongodb.com/manual/sharding/[MongoDB sharded cluster] consists of
 * A separate replica set that acts as the cluster's _configuration server_
 * One or more _routers_ (also called `mongos`) to which clients connect and that routes requests to the appropriate shards
 +
-To use the MongoDB connector with a sharded cluster, configure the connector with the host addresses of the _configuration server_ replica set. When the connector connects to this replica set, it discovers that it is acting as the configuration server for a sharded cluster, discovers the information about each replica set used as a shard in the cluster, and will then start up a separate task to capture the changes from each replica set. If new shards are added to the cluster or existing shards removed, the connector will automatically adjust its tasks accordingly.
+To use the MongoDB connector with a sharded cluster, provide the https://www.mongodb.com/docs/manual/reference/connection-string/[sharded cluster connection string] through the connector's `mongodb.connection.string` property.
+
+[WARNING]
+====
+Previous versions of MongoDB connector required the connector to be configured with the host addresses of the _configuration server_ replica set via the `mongodb.hosts` property. Connection string used by the newer connector versions should include the addresses of Mongo routes (also called `mongos`).
```

Comment:
```suggestion
The `mongodb.connection.string` property replaces the deprecated `mongodb.hosts` property that was used to provide earlier versions of the connector with the host address of the _configuration server_ replica.  
In the current release, use `mongodb.connection.string` to provide the connector with the addresses of MongoDB routes, also known as `mongos`.
```

Reply:
Here was a typo,  the last sentence should read "MongoDB routeRs"